src/custom/state/swap/hooks.ts

```diff
@@ -25,6 +25,7 @@ import { ParsedQs } from 'qs'
 import { DEFAULT_NETWORK_FOR_LISTS } from 'constants/lists'
 import { WETH_LOGO_URI, XDAI_LOGO_URI } from 'constants/index'
 import { WrappedTokenInfo } from '../lists/hooks'
+import { isFeeGreaterThanPriceError } from '../price/utils'
```

Comment:
absolute path?

Reply:
noted in https://github.com/gnosis/cowswap/issues/751